web_poet/rules.py

```diff
@@ -157,23 +157,29 @@ def add_rule(self, rule: ApplyRule) -> None:
         if matched:
             # A common case when a page object subclasses another one with the
             # same URL pattern.
-            pattern_dupes = [
+            pattern_dupes = {
```

Comment:
Because all patterns are equal, it doesn't make much sense to show a list of these patterns; they're the same. Am I missing something here?